plugins/Referrers/API.php

```diff
@@ -365,6 +365,13 @@ public function getSocials($idSite, $period, $date, $segment = false, $expanded
 
         $dataTable = Archive::createDataTableFromArchive(Archiver::SOCIAL_NETWORKS_RECORD_NAME, $idSite, $period, $date, $segment, $expanded, $flat);
 
+        $dataTable->filter('GroupBy', ['label', function($value) {
```

Comment:
Could we do this only if one or the other label exists? Typically the GroupBy filter is quite slow, and checking if a label exists in the report is really fast usually. 

Not too important but could be worth thinking about it.

Reply:
updated the branch